pontoon/sync/formats/l20n.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import absolute_import
+
+import codecs
+import copy
+import logging
+
+from l20n.format.parser import L20nParser
+from l20n.format.serializer import Serializer
+from l20n.format import ast
+
+from pontoon.sync import SyncError
+from pontoon.sync.formats.base import ParsedResource
+from pontoon.sync.vcs.models import VCSTranslation
+
+log = logging.getLogger(__name__)
+
+
+class L20NEntity(VCSTranslation):
+    """
+    Represents entities in l20n (without its attributes).
+    """
+    def __init__(self, type, key, source_string, source_string_plural, strings, comments=None, order=None):
+        self.type = type
+        super(L20NEntity, self).__init__(
+            key=key,
+            source_string=source_string,
+            source_string_plural=source_string_plural,
+            strings=strings,
+            comments=comments or [],
+            fuzzy=False,
+            order=order,
+        )
+
+    def __repr__(self):
+        return '<L20NEntity {type}:{key}>'.format(type=self.type, key=self.key.encode('utf-8'))
+
+
+class L20NResource(ParsedResource):
+    def __init__(self, path, locale, source_resource=None):
+        self.path = path
+        self.locale = locale
+        self.entities = {}
+        self.source_resource = source_resource
+        if source_resource:
```

Comment:
Add blank line and a comment **above** this line:
`# Copy entities from the source_resource if it's available.`